packages/patterns-react/src/patterns/CardSection/CardSection.js

```diff
@@ -67,25 +67,36 @@ const CardSection = ({ title, cards, theme }) => {
       className={classNames(`${prefix}--card-section`, setTheme(theme))}>
       <div className={`${prefix}--card-section__container`}>
         <div className={`${prefix}--card-section__row`}>
-          <div className={`${prefix}--card-section__col`}>
+          <div className={`${prefix}--card-section__col--left`}>
             <h2 className={`${prefix}--card-section__title`}>{title}</h2>
+          </div>
+          <div className={`${prefix}--card-section__col--right`}>
             <div
-              className={`${prefix}--card-section__cards`}
+              className={classNames(
+                `${prefix}--card-section__cards`,
+                `${prefix}--grid--condensed`
+              )}
               ref={containerRef}>
-              {cards.map((card, index) => {
-                return (
-                  <CardLink
-                    key={index}
-                    imgSrc={card.imgSrc}
-                    altText={card.altText}
-                    title={card.title}
-                    content={card.copy}
-                    href={card.link.href}
-                    target={card.link.target}
-                    icon={<ArrowRight20 />}
-                  />
-                );
-              })}
+              <div className={`${prefix}--card-section__cards__container`}>
+                <div className={`${prefix}--card-section__cards__row`}>
```

Comment:
So we talked a little about this on the call this morning. I think this can be simplified. Now that we have it working let's try to get rid of what we don't need. I think reducing some of this will also resolve some of the alignment issues I'm seeing.

Take a look at the following...

```jsx
<div className= className={`${prefix}--card-section__container`}>
  <div className={`${prefix}--card-section__row`}>
    <h2 className={`${prefix}--card-section__title`}>{title}</h2>
    <div className={`${prefix}--card-section__cards`}>
      <div className={`${prefix}--card-section__cards ${prefix}--row--condensed`}>
      {cards.map((card, index) => {
        return (
          <div className={`${prefix}--card-section__cards__col`}>
            <CardLink />
          </div>
        );
      })}
    </div>
  </div>
</div>
```

```scss
.card-section {
  @include carbon--make-container;

  &__row {
    @include carbon--make-row;
  }

  &__title {
    @include carbon--make-col-ready;
    @include carbon--make-col(4, 16);
    // add your breakpoints and spans.
  }

  &__cards {
    @include carbon--make-col-ready;
    @include carbon--make-col(12, 16);
    // add your breakpoints and spans.

    &__row {
      @include carbon--make-row;
    }
    &__col {
      @include carbon--make-col-ready;
      @include carbon--make-col(1, 3);
      // add your breakpoints and spans.
    }
  }
}
```

Notice how `col--left` and `col--right` was moved onto `title` and `cards`, and how we removed `cards__container`.